torchbench.py

```diff
@@ -329,6 +332,47 @@ def speedup_experiment(args, model_iter_fn, model, example_inputs):
     )
     return format_speedup(speedup, pvalue, is_correct=is_correct)
 
+def quant_experiment(args, model_iter_fn, float_model, quant_model, example_inputs):
+    """
+    Measure speedups over eager using the autotuning inference backend.  To use this:
+        1) First run once to record graphs that need autotuning
+        2) Next run ./autotune.py to select the right backend for each recorded graph
+        3) Finally, run this target again to measure speedups
+
+    Writes to ./speedups.csv
+    """
+    timings = np.zeros((args.repeat, 2), np.float64)
+    # if we randomize the input, we should also check the result is correct
+    should_check_result = should_randomize_input = args.randomize_input
+    is_correct = True
+
+    for rep in range(args.repeat):
+        inputs = (
+            randomize_input(copy.deepcopy(example_inputs))
+            if should_randomize_input
+            else example_inputs
+        )
+
+        # interleave the runs to handle frequency scaling and load changes
+        with torchdynamo.run():
```

Comment:
I think the issue might be this line here.

TorchDynamo cache on the code object level, so it possible some quantized compiled code is leaking into your baseline.